bin/cml/runner.js

```diff
@@ -276,15 +276,13 @@ const runLocal = async (opts) => {
         if (cml.driver === 'github') {
           const job = await cml.runnerJob({ runnerId: RUNNER_ID });
 
-          if (!job && !idle) {
+          if (!job && idle) {
```

Comment:
This is not correct.

If not job being runned by the runner and not yet idle (jobs in internal queue) runner should be idle
Allowing to

Reply:
Then if we have successfully parsed the job finish and we have the internal queue empty (idle) but the job still exists in the CI means that we should not wait another round and confirm again if job is still in the CI

The issue might not be in our side but be one of the errors already known in GH see #689

Reply:
What i pretend to say is that job is empty when it should not. We could add the idle from the api as one of the trials. But I actually found it failing while job check never failed for me

Reply:
Thanks! The original pull request was completely misguided.